app/models.py

```diff
@@ -10,7 +10,7 @@ class Member(models.Model):
     email = models.CharField(max_length=60)
     institution = models.CharField(max_length=100, null=True)
     designation = models.CharField(max_length=200)
-    image = models.FileField(null=True, default=None)
```

Comment:
default=None could be useful in showing a default "avatar" in case image isn't there.

Reply:
Default images will be stored in the static folder.
Application server files will be served in a different route. So even if we put the static path it can't be served like this.